crates/core/main.rs

```diff
@@ -68,11 +68,20 @@ fn try_main(args: Args) -> Result<()> {
         process::exit(0)
     } else if messages::errored() {
         process::exit(2)
+    } else if !matched {
```

Comment:
I wasn't sure if the last `else` branch can be reached in any case that isn't the "nothing searched" case, so I made a new `else if`.

Reply:
Yes, the `else` branch is only reached when `matched` is false.